interface/reports/inventory_list.php

```diff
@@ -26,46 +26,662 @@
+        echo csvEscape($row['active'] ? xl('Yes') : xl('No')) . ',';
+        echo csvEscape($row['consumable'] ? xl('Yes') : xl('No')) . ',';
+        echo csvEscape($drug_form) . ',';
+        if ($form_details) {
+            echo csvEscape($row['facname'])                   . ',';
+            if ($form_details == 2) { // warehouse details {
+                echo csvEscape($row['title'])                 . ',';
+            }
+            echo csvEscape($row['pw_min_level'])              . ',';
+            echo csvEscape($row['pw_max_level'])              . ',';
+        } else {
+            echo csvEscape($row['reorder_point'])             . ',';
+            echo csvEscape($row['max_level'])                 . ',';
+        }
+        echo csvEscape($row['on_hand'])                       . ',';
+        echo csvEscape($zerodays)                             . ',';
+        echo csvEscape($monthly)                              . ',';
+        echo csvEscape($stock_months)                         . ',';
+        echo csvEscape($reorder_qty)                          . ',';
+        echo csvEscape($warnings)                             . '';
+        echo "\n";
+    } else { // end exporting
+        echo " <tr class='detail' bgcolor='$bgcolor'>\n";
+        if ($drug_id == $wrl_last_drug_id) {
+            echo "  <td colspan='6'>&nbsp;</td>\n";
+        } else {
+            echo "  <td>" . text($row['name'])                            . "</td>\n";
+            echo "  <td>" . text($relcodes)                               . "</td>\n";
+            echo "  <td>" . text($row['ndc_number'])                      . "</td>\n";
+            echo "  <td>" . ($row['active'] ? xlt('Yes') : xlt('No'))     . "</td>\n";
+            echo "  <td>" . ($row['consumable'] ? xlt('Yes') : xlt('No')) . "</td>\n";
+            echo "  <td>" . text($drug_form)                              . "</td>\n";
+        }
+        if ($form_details) {
+            echo "  <td>" . text($row['facname'])                         . "</td>\n";
+            if ($form_details == 2) { // warehouse details {
+                echo "  <td>" . text($row['title'])                         . "</td>\n";
+            }
+            echo "  <td align='right'>" . text($row['pw_min_level'])      . "</td>\n";
+            echo "  <td align='right'>" . text($row['pw_max_level'])      . "</td>\n";
+        } else {
+            echo "  <td align='right'>" . text($row['reorder_point'])     . "</td>\n";
+            echo "  <td align='right'>" . text($row['max_level'])         . "</td>\n";
+        }
+        echo "  <td align='right'>" . text($row['on_hand'])               . "</td>\n";
+        echo "  <td align='right'>" . text($zerodays)                     . "</td>\n";
+        // Do not text() these next 2 lines.
+        echo "  <td align='right'>" . $monthly                            . "</td>\n";
+        echo "  <td align='right'>" . $stock_months                       . "</td>\n";
```

Comment:
text() 2 lines above

Reply:
See comment just above those lines. This is because the values might contain "&nbsp". Otherwise they are guaranteed to be numeric.